DTAdmin/CommonNetworkManager.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NetworkManager.swift
+//  DTAdmin
+//
+//  Created by mac6 on 10/19/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import Foundation
+
+struct Keys {
+    static let ID_KEY = "id"
+    static let USER_NAME_KEY = "username"
```

Comment:
Use `static let username` for it. Consult with https://github.com/raywenderlich/swift-style-guide#constants for style guide for constants